Libraries/WebSocket/WebSocket.android.js

```diff
@@ -11,28 +11,99 @@
  */
 'use strict';
 
+var RCTDeviceEventEmitter = require('RCTDeviceEventEmitter');
+var RCTWebSocketManager = require('NativeModules').WebSocketManager;
```

Comment:
WebSocket.android/iOS.js should be replaced with just WebSocket.js, or they should be small files that inject their differences into WebSocketBase.js.